docs/misc/open-source-development.md

```diff
@@ -0,0 +1 @@
+# Open source libraries 
```

Comment:
What's this file for?

Reply:
whoops! sorry I've thought to move it first into a separate document